crates/context_aware_config/src/api/config/types.rs

```diff
@@ -1,20 +0,0 @@
-use std::collections::HashMap;
-
-use serde::{Deserialize, Serialize};
-use serde_json::{Map, Value};
-use superposition_types::{Condition, Overrides};
-
-#[derive(Serialize, Deserialize)]
-pub struct Config {
-    pub contexts: Vec<Context>,
-    pub overrides: HashMap<String, Overrides>,
-    pub default_configs: Map<String, Value>,
-}
-
-#[derive(Serialize, Clone, Deserialize)]
-pub struct Context {
-    pub id: String,
-    pub condition: Condition,
-    pub priority: i32,
-    pub override_with_keys: [String; 1],
-}
```

Comment:
INFO: Not needed anymore (moved as a core type) [here](https://github.com/juspay/superposition/pull/264/files#diff-61d7d423a4c1cc86aa26179f4586efff51e9d8dc8a0d414868d820073e622de6R156-R175)